ppgan/models/generators/deoldify.py

```diff
@@ -57,6 +57,7 @@ class Deoldify(SequentialEx):
     def __init__(self,
                  encoder,
                  n_classes,
+                 model_type='stable',
```

Comment:
please also update app.md and https://github.com/PaddlePaddle/PaddleGAN/blob/master/docs/zh_CN/tutorials/video_restore.md

Reply:
done, thanks